react-exercises/grocery-list-react/src/components/add-grocery/add-grocery.component.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './add-grocery.style.css'
+export const AddGroceryForm = ({handleSubmit, handleChange}) => {
+  return (
+    <div>
+       <form onSubmit={handleSubmit}>
+    
+          <input type="text"  placeholder="Enter Item name" onChange={handleChange} />
+  
+          <input type="submit" value="Add Item"/>
```

Comment:
The new lines here are not required as such
```suggestion
          <input type="text"  placeholder="Enter Item name" onChange={handleChange} />
          <input type="submit" value="Add Item"/>
```